src/components/StaticThought.tsx

```diff
@@ -180,7 +182,7 @@ const StaticThought = ({
           // left, top are eyeballed for different font sizes
           <HomeIcon style={{ position: 'relative', left: fontSize - 14, top: fontSize / 4 - 1 }} />
         ) : isDivider(value) ? (
-          <Divider path={simplePathLive} />
+          <Divider cssRaw={css.raw({ borderColor: 'rgb(204, 204, 204)' })} path={simplePathLive} />
```

Comment:
Can we move the border color into the Divider component itself? Every Divider should have this border color applied from what I can tell, so there is no reason to parameterize it.